clients/js/package-lock.json

```diff
@@ -8,7 +8,7 @@
       "name": "wormhole-client",
       "version": "1.0.0",
       "dependencies": {
-        "@certusone/wormhole-sdk": "^0.3.0",
```

Comment:
I think you missed recommitting the package-lock in the last commit

Reply:
Fixed.